diff -ruN asus-touchpad-numpad-driver-main.orig/asus_touchpad.py asus-touchpad-numpad-driver-main/asus_touchpad.py
--- asus-touchpad-numpad-driver-main.orig/asus_touchpad.py	2021-11-17 17:25:34.000000000 +0100
+++ asus-touchpad-numpad-driver-main/asus_touchpad.py	2022-02-28 02:04:45.993535299 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 
 import importlib
 import logging
diff -ruN asus-touchpad-numpad-driver-main.orig/asus_touchpad.service asus-touchpad-numpad-driver-main/asus_touchpad.service
--- asus-touchpad-numpad-driver-main.orig/asus_touchpad.service	2021-11-17 17:25:34.000000000 +0100
+++ asus-touchpad-numpad-driver-main/asus_touchpad.service	2022-02-28 02:08:24.917365438 +0100
@@ -3,14 +3,16 @@
 
 [Service]
 Type=simple
-ExecStart=/usr/share/asus_touchpad_numpad-driver/asus_touchpad.py $LAYOUT $PERCENTAGE_KEY
+ExecStart=/usr/bin/asus_touchpad.py $LAYOUT $PERCENTAGE_KEY
 StandardInput=tty-force
-StandardOutput=/var/log/asus_touchpad_numpad-driver/error.log
-StandardError=/var/log/asus_touchpad_numpad-driver/error.log
+StandardOutput=/var/log/asus-touchpad-numpad-driver/error.log
+StandardError=/var/log/asus-touchpad-numpad-driver/error.log
 TimeoutSec=5
 Restart=on-failure
 # For some operating systems with boot failure, adding this following property might resolve the issue  
 # ExecStartPre=/bin/sleep 2
+ExecStartPre=/usr/share/asus-touchpad-numpad-driver/loadModules.sh
+ExecStop=/usr/share/asus-touchpad-numpad-driver/touchReset.sh
 
 [Install]
 WantedBy=default.target
diff -ruN asus-touchpad-numpad-driver-main.orig/tests/test_brightness.py asus-touchpad-numpad-driver-main/tests/test_brightness.py
--- asus-touchpad-numpad-driver-main.orig/tests/test_brightness.py	2021-11-17 17:25:34.000000000 +0100
+++ asus-touchpad-numpad-driver-main/tests/test_brightness.py	2022-02-28 02:04:45.993535299 +0100
@@ -1,4 +1,4 @@
-#!/bin/python3
+#!/usr/bin/python3
 
 import re
 import os
