diff -ruN asus-touchpad-numpad-driver-main.orig/asus_touchpad.service asus-touchpad-numpad-driver-main/asus_touchpad.service
--- asus-touchpad-numpad-driver-main.orig/asus_touchpad.service	2021-11-17 17:25:34.000000000 +0100
+++ asus-touchpad-numpad-driver-main/asus_touchpad.service	2022-02-24 22:08:30.914786344 +0100
@@ -10,7 +10,8 @@
 TimeoutSec=5
 Restart=on-failure
 # For some operating systems with boot failure, adding this following property might resolve the issue  
-# ExecStartPre=/bin/sleep 2
+ExecStartPre=/usr/share/asus_touchpad_numpad-driver/loadModules.sh
+ExecStop=/usr/share/asus_touchpad_numpad-driver/touchReset.sh
 
 [Install]
 WantedBy=default.target
